smartHome/src/main/java/com/cnu2016/assignment02/EventGenerator.java

```diff
@@ -0,0 +1,64 @@
+package com.cnu2016.assignment02;
+import java.util.*;
+import java.io.*;
+public class EventGenerator {
+    static class Job {
+        int ID, timeStamp;
+        boolean state;
+        public Job (int ID, int timeStamp, boolean state) {
+            this.ID = ID;
+            this.timeStamp = timeStamp;
+            this.state = state;
+        }
+    };
+    public static ArrayList<Job> getSortedJobs(String filename) throws IOException, BadScheduleException {
```

Comment:
different methods should be used for different use-cases
use different methods for reading and sorting, say readEvents() and processEvents()